### ISSUE TITLE
Storybook Mobile and HTML add-ons have stopped working

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#1884]
## Overview

Since Renovate updated the Storybook monorepo to `6.5.9` (#1843), the following add-ons have stopped working:

- [Storybook Mobile](https://github.com/aholachek/storybook-mobile)
- [Storybook HTML](https://github.com/whitespace-se/storybook-addon-html)

## Steps to reproduce

1. Open any story in "canvas" view.
2. Observe whether or not "Mobile" and "HTML" tabs are next to "Controls" and "Accessibility."

- ✅ [Example with Storybook 6.4.22](https://deploy-preview-1758--cloudfour-patterns.netlify.app/?path=/story/components-heading--example)
- ❌ [Example with Storybook 6.5.9](https://deploy-preview-1843--cloudfour-patterns.netlify.app/?path=/story/components-heading--example)
- ❌ [Example with latest](https://v-next--cloudfour-patterns.netlify.app/?path=/story/components-heading--example)

---

- See #1717 
- See https://github.com/aholachek/storybook-mobile/issues/35
- See https://github.com/storybookjs/storybook/issues/14489

**Comment by tylersticka** [comment]
@spaceninja FYI, some acceptable outcomes of this issue for me are:

- Determine this issue is blocked by dependency updates.
- Determine that there is a reasonable, working alternative we could use instead.
- Suggest removing the dependency for now.

My main goal in creating this issue was to get 👀 on it to determine our best next steps.

**Comment by tylersticka** [comment]
FYI, removing this from the project backlog because it is not a launch blocker.

**Comment by spaceninja** [comment]
Note that we disabled these two plugins (though they're still installed as dependencies so we can monitor updates) in #1954

**Comment by yannbf** [comment]
@spaceninja regarding Storybook Mobile, version 1.0.1 has been released which should be working now with Storybook 6.5.
It also seems to live in a different repo: https://github.com/stevensacks/storybook-mobile-addon

**Comment by spaceninja** [comment]
Closing: one of these is fixed and the other cannot be.

**Storybook HTML is back.** #2416 restored `@whitespace/storybook-addon-html` at v9, and it is in `.storybook/main.mjs` today. It was verified rendering markup in the panel on that PR's deploy preview.

**Storybook Mobile is not coming back.** It is still at **1.0.1** and peers on `react: ^17.0.2`, while Storybook 10 runs React 19. The package has not shipped a release in years, so this is not a wait-and-see — there is nothing upstream to wait for. If we want that capability again it would mean finding a different addon or doing without, which is a new decision rather than this bug.

Nothing actionable is left here. If the mobile-hints capability is still wanted, that is worth its own issue framed as a choice rather than a regression.
